### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by MuriloRoque**
Dear TSE, hello! :wave:  

Thank you for reviewing my project :grin:

Here's what I did in this milestone:

- Used flex display mode for App, Display and ButtonPanel components;
- Set the Calculator width to 700px;
- Styled the Display component according to the suggestions;
- Styled the ButtonPanel component according to the suggestions;
- Styled the Button component according to the suggestions and more:
   - Added an effect when hovering and clicking the button.
- Passed “color” and “wide” props to the Button component and styled according to the suggestions.

### CODE REVIEW

**Review by Torres-ssf — APPROVED** [review]
### Milestone approved :tada: 
Hi there @MuriloRoque 
Nice job on this milestone :+1: 
There is just a minor change that we need to make, and the reason is that it's a requirement. Since it's just a small change I **approved this PR**, but I trust you to make this change before merging the branch. 

- [ ] The prop color should have a defaulProps value of "orange" when it's not passed to the button component.
Screenshot from 2020-07-13 10-50-58


Good luck with the next milestone :+1: 
Happy Coding :computer: 
**Sergio Torres**
[Twitter](https://twitter.com/torres_ssf) [LinkedIn](https://www.linkedin.com/in/torres-ssf/) [Portifolio](https://www.torres-ssf.com/)

**Comment by MuriloRoque** [comment]
Alright @Torres-ssf , leave it to me, thank you again! 😁

**Pull request merged**
